styles/_library.scss

```diff
@@ -1,17 +1,51 @@
 //vars
 //mainColor
 
-$bgColor: #0d0d0d;
-$text: #fff;
+$primary-100:#73A5FF;
+$primary-300: #4C8DFF;
+$primary-500: #397DF6;
+$primary-700: #2F68CC;
+$primary-900: #234E99;
+$success-100: #80FFBF;
+$success-300: #22E584;
+$success-500: #14CC70;
+$success-700: #0F9954;
+$success-900: #0A6638;
+$danger-100: #FF8099;
+$danger-300: #F23D61;
+$danger-700: #990F2B;
+$danger-900: #660A1D;
+$warning-100: #FFD073;
+$warning-300: #E5AC39;
+$warning-500: #D99000;
+$warning-700: #960;
+$warning-900: #640;
+$light-100: #FFF;
+$light-300: #F7FBFF;
+$light-500: #EDF3FA;
+$light-700: #D5DAE0;
+$light-900: #8D9094;
+$dark-100: #4C4C4C;
+$dark-300: #333;
+$dark-500: #171717;
+$dark-700: #0D0D0D;
+$dark-900: #000;
 
 @mixin btn {
```

Comment:
that`s not the place for mixin`s ,thats the place for vars . Create another place where mixin`s for component`s will be